drivers/Makefile

```diff
@@ -1,7 +1,9 @@
 # Makefile for lwip drivers to be included
 
-# TODO: make every driver a separate component?
-ifneq ($(EPHY_KSZ8081RND),)
+ifeq ($(EPHY_KSZ8081),RNA)
+  LOCAL_CFLAGS := -DEPHY_KSZ8081RNA
+endif
+ifeq ($(EPHY_KSZ8081),RND)
```

Comment:
Perhaps it would be better to remove RND/RNA compile time distinction, as we now support dynamic detection anyway?

Reply:
We support dynamic **selection** by custom hook. Dynamic detection proved to be unreliable in some circumstances (software reboot, cable in the port, etc.) so finally it's not implemented (maybe after some more experimentation it could be done more consistently). The `do {} while (0)` is a remnant of `do {} while (ephy_setAltConfig(cfg++) > 0)`.

We sometimes still would be able to force one of the possibilites at a compile time (eg. during production when board revision is not yet burned into fuses). That's why I intentionally left it as an env variable (not a `board_config.h` const) - to be able to easily force it from command line when building.